pyads/ads.py

```diff
@@ -713,11 +720,16 @@ def get_all_symbols(self) -> List[AdsSymbol]:
                 comment_end_ptr = comment_start_ptr + comment_length
 
                 name = decode_ads(symbol_list_msg[name_start_ptr:name_end_ptr])
-                symtype = decode_ads(symbol_list_msg[type_start_ptr:type_end_ptr])
-                comment = decode_ads(symbol_list_msg[comment_start_ptr:comment_end_ptr])
+                symbol_type = decode_ads(symbol_list_msg[
+                                   type_start_ptr:type_end_ptr])
+                comment = decode_ads(symbol_list_msg[
+                                     comment_start_ptr:comment_end_ptr])
```

Comment:
Multi-line statements should be aligned properly following [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation). In this case putting it on a single line is even more readable.

```suggestion
                symbol_type = decode_ads(symbol_list_msg[type_start_ptr:type_end_ptr])
                comment = decode_ads(symbol_list_msg[comment_start_ptr:comment_end_ptr])
```